docs/content/getting-started/perform-check.mdx

```diff
@@ -163,6 +164,49 @@ To check whether user `user:anne` has relationship `reader` with object `documen
 
 The result's `allowed` field will return `true` if the relationship exists and `false` if the relationship does not exist.
 
+### 03. Calling Batch Check API
+
+If you want to check multiple user-object-relationship combinations in a single request, you can use the [Batch Check](/api/service#Relationship%20Queries/BatchCheck) API endpoint. Batching authorization checks together in a single request significantly reduces overall network latency.    
+
+The Batch Check endpoint requires a `correlation_id` parameter for each check. The `correlation_id` is used to "correlate" the check respones with the checks sent in the request, since `tuple_keys` and `contextual_tuples` are not returned in the response on purpuse to reduce data transfer to improve network latency. A `correlation_id` can be composed of any string of alphanumeric characters or dashes between 1-36 characters in length. 
+This means you can use:
+- simple iterating integers `1,2,3,etc`
+- UUID `e5fe049b-f252-40b3-b795-fe485d588279`
+- ULID `01JBMD9YG0XH3B4GVA8A9D2PSN`
+- or some other unique string
+
+But each `correlation_id` within a request must be unique.
+
+*Coming Soon*: If you are using one of our SDKs, the `correlation_id` is inserted for you by default and automatically correlates the `allowed` response with the proper `tuple_key`.
```

Comment:
Maybe we change this into a `Note:` rather than coming soon? The correlation ID wont be relevant until we roll out the new endpoint into the SDKs.